src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -31,7 +31,7 @@ class HtmlDumper extends CliDumper
     protected $headerIsDumped = false;
     protected $lastDepth = -1;
     protected $styles = array(
-        'default' => 'background-color:#18171B; color:#FF8400; line-height:1.2em; font:12px Menlo, Monaco, Consolas, monospace; word-wrap: break-word; white-space: pre-wrap; position:relative; z-index:100000',
+        'default' => 'background-color:#18171B; color:#FF8400; line-height:1.2em; font:12px Menlo, Monaco, Consolas, monospace; word-wrap: break-word; white-space: pre-wrap; position:relative; z-index:100000; display: block; padding: 5px; word-break: normal',
```

Comment:
Removing `word-wrap: break-word` will lead to #13487 issue.

Removing `word-break: normal` will lead to the issue targeted within this PR if some css framework or whatever set `word-break` to `break-all` for `pre` tags.